texttospeech/api/TextToSpeech.Samples/ListVoices.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START tts_list_voices]
+
+using System.Collections.Generic;
+using System.Linq;
+using Google.Cloud.TextToSpeech.V1;
+
+public class ListVoicesSample
+{
+    public List<Voice> ListVoices(string desiredLanguageCode = "")
```

Comment:
```suggestion
    public IEnumerable<Voice> ListVoices(string desiredLanguageCode = "en-US")
```
Or other desired language code.

Reply:
![image](https://user-images.githubusercontent.com/59259370/91726191-7905f000-ebbd-11ea-9f25-a07ecd65cc21.png)

Can't Have IEnumerable,
As In Test We Won't Be Able To Convert It Back To List

Reply:
Yes In Test Please add `ToList()`

Reply:
ToList Not Available

Reply:
I'll Have To Cast It Manually
Like So
![image](https://user-images.githubusercontent.com/59259370/91729123-b53b4f80-ebc1-11ea-8aca-e2ace7f7c4fa.png)